### PR TITLE
Add K8sGpt as Default Application

### PR DESCRIPTION
**Pull request by mfahlandt**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
#12994

**What type of PR is this?**

/kind feature

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Add k8sgpt operator to the Default Application Catalogue
```

**Documentation**:
```documentation
https://github.com/kubermatic/docs/pull/1610
```

### CODE REVIEW

**Comment by scheeles** [comment]
Can we add the CLI also to the Webterminal

**Comment by mfahlandt** [comment]
I guess we can add it to the image that gets started - i would have it in a seperate pr / feature

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve 
/retest

File: pkg/ee/default-application-catalog/applicationdefinitions/k8sgpt-app.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#                Kubermatic Enterprise Read-Only License
+#                       Version 1.0 ("KERO-1.0”)
+#                   Copyright © 2023 Kubermatic GmbH
```
Comment (ahmedwaleedmalik) [inline pkg/ee/default-application-catalog/applicationdefinitions/k8sgpt-app.yaml]: `s/2023/2024`

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

**Pull request merged**